closed/src/java.base/share/native/libjncrypto/NativeCrypto_md.h

```diff
@@ -24,8 +24,9 @@
 
 #ifndef NATIVECRYPTO_MD_H
 #define NATIVECRYPTO_MD_H
+#include <jni.h>
```

Comment:
A blank line would be welcome before this line.

Reply:
Edited.